tools/telemetry-generator/package.json

```diff
@@ -35,13 +35,20 @@
 		"applicationinsights": "^2.4.1"
 	},
 	"devDependencies": {
+		"@fluid-internal/mocha-test-setup": "~2.0.5",
 		"@fluidframework/build-common": "^2.0.1",
 		"@fluidframework/eslint-config-fluid": "^5.2.0",
+		"@types/mocha": "^9.1.1",
 		"@types/node": "^14.18.0",
+		"@types/sinon": "^17.0.3",
 		"concurrently": "^8.2.1",
 		"eslint": "~8.55.0",
+		"mocha": "^10.7.3",
+		"mocha-json-output-reporter": "^2.0.1",
```

Comment:
Hmm Tony removed the json output reporter from the repo recently, but I think it's still present in mocha-test-setup 2.3.1, so we probably need it here for now... I'll create a task to clean this up later.

Reply:
Let me try remove this one and see what happened

Reply:
Removed it here